packages/suite-desktop/src/libs/store.ts

```diff
@@ -54,15 +56,17 @@ export class Store {
     }
 
     public getTorSettings() {
-        return this.store.get('torSettings', {
-            running: false,
-            address: '',
-        });
+        return this.store.get('torSettings', { running: false, host: '', port: NaN });
```

Comment:
```suggestion
        return this.store.get('torSettings', { running: false, host: '', port: 9150 });
```
I would rather use default tor port instead of `NaN`

Reply:
Even better maybe?
https://github.com/trezor/trezor-suite/pull/8271/commits/1b9a2e25b3b57b45321a2ae91de13006d95a6603 + https://github.com/trezor/trezor-suite/pull/8271/commits/8bad15d18911b55ff6aa2a382f27f1c9b3dc0ceb